src/pivot-table/components/cells/DimensionTitleCell.tsx

```diff
@@ -75,10 +85,18 @@ const DimensionTitleCell = ({
     [cell, isDim],
   );
 
-  const sortFromMenu = async (evt: React.MouseEvent, newSortDirection: SortDirection) => {
-    evt.stopPropagation();
-    await changeSortOrder(headerData, newSortDirection);
-  };
+  const sortRelatedArgs = useMemo(() => {
```

Comment:
Might be a bit overkill to wrap these props in `useMemo`. I wrapped the `HeadCellMenu` component in the util repo with `React.memo`. Think I need some more testing to ensure that it actually reduces the number of commited renders.

Reply:
+1

Reply:
It did not change the number of renders. I've now removed useMemo.